ipa-core/src/cli/crypto/hybrid_encrypt.rs

```diff
@@ -52,15 +52,38 @@ pub struct HybridEncryptArgs {
     /// Path to helper network configuration file
     #[arg(long)]
     network: PathBuf,
+    /// a flag to produce length delimited binary instead of newline delimited hex
+    #[arg(long)]
+    length_delimited: bool,
```

Comment:
Instead of a boolean argument it might be clearer to have descriptive options (something like the `FileFormat` choices -- possibly clap can even be taught to parse into `FileFormat`).

Reply:
I generally think of [docopt](http://docopt.org) as the canonical way to design CLIs, and this would probably fit as a mutually exclusive element better than an element with an arg. Clap probably has a way of defining mutually exclusive elements (and as you say, clap could likely be taught to coerce that into the enum.) This would be even more important if there were more than two options. 

I likely won't get to this before I'm out, so up to @andyleiserson and @akoshelev if you want to refactor before merging.